tutor/commands/config.py

```diff
@@ -145,10 +152,13 @@ def save(
     remove_vars: list[tuple[str, t.Any]],
     unset_vars: list[str],
     env_only: bool,
+    clean_env: bool,
 ) -> None:
     config = tutor_config.load_minimal(context.root)
     if interactive:
         interactive_config.ask_questions(config)
+    if clean_env:
```

Comment:
Point 4 https://github.com/overhangio/tutor/issues/967#issuecomment-1903535377 does not seem to be there.

Reply:
You're right. I forgot about it.

Reply:
I'm not sure if this is the best way to implement it in interactive mode, but I'm open to any suggestions.